configs/csl/metafile.yml

```diff
@@ -0,0 +1,27 @@
+Collections:
+- Name: CSL
+  Metadata:
+    Training Data: DOTAv1.0
+    Training Techniques:
+      - SGD with Momentum
+      - Weight Decay
+    Training Resources: 1x GeForce GTX 2080 Ti
+    Architecture:
+      - ResNet
+  Paper:
+    URL: None
+    Title: 'Arbitrary-Oriented Object Detection with Circular Smooth Label'
+  README: configs/csl/README.md
```

Comment:
Paper URL and README are incorrect. And need a Code field.

Reply:
We previously referenced MMPose's metafile.yml setting, so there is no code field. Maybe, we could propose another PR later to modify it uniformly.